modules/core/src/main/java/org/apache/ignite/cache/store/jdbc/CacheJdbcBlobStore.java

```diff
@@ -206,6 +213,38 @@
             log.debug("Transaction ended [xid=" + tx.xid() + ", commit=" + commit + ']');
     }
 
+    /** {@inheritDoc} */
+    @Override public void loadCache(IgniteBiInClosure<K, V> clo, Object... args) {
+        init();
+
+        Transaction tx = transaction();
+
+        if (log.isDebugEnabled())
+            log.debug("Store load cache");
+
+        Connection conn = null;
+
+        PreparedStatement stmt = null;
+
+        try {
+            conn = connection(tx);
+
+            stmt = conn.prepareStatement(loadCacheQry);
```

Comment:
if large resultSet will be loaded I suggest to add fetchSize parameter: 10000 for example